_includes/scripts.html

```diff
@@ -25,4 +25,34 @@
     });
   });
 </script>
+
+
+<script>
+  $(document).ready(function(){
+    // Add smooth scrolling to all links
+    $(".nav-link").on('click', function(event) {
```

Comment:
You've made `.nav-link` both a class for functionality, and a style. Could you make two classes, instead? We need to add this slow-scrolling functionality to the links in the footer, too.

Reply:
so maybe it could be named 'slow-scrolling'? so that it can be applied to the footer too?

Reply:
That works!